references/detection/eval.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
```

Comment:
this file is completely unrelated to the windows build, right? I think it's a good idea to have it in the repo, but as a separate PR so that we can focus on how to better illustrate this.